README.md

```diff
@@ -172,14 +172,18 @@ You can alter or add localized strings in `/src/Strings.ts`:
 
 Behavioral customization will require changing the TypeScript files in `/src`. A full description of the architecture of WebChat is beyond the scope of this document, but here are a few starters:
 
-### Architecture
+#### Architecture
```

Comment:
The previous header was 3 levels. Given that this paragraph ended with a colon, it seemed like it should be a sub-section. Thoughts?